draft-ietf-tls-exported-authenticator.md

```diff
@@ -331,7 +345,8 @@ The "validate" API takes as input:
 * an optional authenticator request
 * an authenticator
 
-It returns the certificate chain and extensions.
+It returns the certificate chain and extensions for a non-empty authenticator,
+and a status to indicate whether the authenticator is valid or not.
```

Comment:
"It returns the certificate chain and extensions and a status to indicate whether the authenticator is valid or not.  If the authenticator was empty - that is, it did not contain a certificate - the certificate chain will contain no certificates."

This is silent on whether any extensions might be included intentionally.  I think that we might want to allow them.  Maybe they can be used to signal something in future, like hints about what sort of request might be successful.

Reply:
What do you mean by "included intentionally"?

Reply:
Sorry, unclear.  If the rejection wanted to include extensions, then I don't see why we should prevent that.  None of our current extensions are relevant, but some future one might be.  That's all.